### PR TITLE
change the task window

### PR DESCRIPTION
**Pull request by CarmenMartinez**

### CODE REVIEW

File: src/gui/controllers/TaskController.java [hunk 1]
```diff
@@ -18,5 +22,25 @@
...
+	@FXML public void accept(ActionEvent event) {
+		Object userData = textFieldTitle.getScene().getWindow().getUserData();
+		if (isValidData() && userData != null && userData instanceof TasksHandler) {
+			try {
+				((TasksHandler) userData).addTask(new Task(textFieldTitle.getText(),textAreaBody.getText()));
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+		}
+		Utils.closeWindow(event);
```
Comment (AlexisGlez) [inline src/gui/controllers/TaskController.java]: I think this line should be executed only if the task is saved.
Move this line between line 30 and 31.

File: src/gui/controllers/TaskController.java [hunk 2]
```diff
@@ -18,5 +22,25 @@
...
+	}
+
+	@FXML public void cancel(ActionEvent event) {
+		Utils.closeWindow(event);
+	}
+	
+	private boolean isValidData() {
+		return textFieldTitle.getText().length() > 0;
```
Comment (AlexisGlez) [inline src/gui/controllers/TaskController.java]: Why are you not validating if the textAreaBody has content? Are we going to let users save tasks with just a title?

**Review by AlexisGlez — CHANGES_REQUESTED** [review]

File: src/gui/controllers/TaskController.java (hunk 2 above)
Comment (CarmenMartinez) [reply src/gui/controllers/TaskController.java]: You're right, I forgot to validate the textarea.
Actually, we have to validate the date too, if the task is a reminder.

File: src/gui/controllers/TaskController.java (hunk 1 above)
Comment (CarmenMartinez) [reply src/gui/controllers/TaskController.java]: Done

File: src/gui/controllers/TaskController.java (hunk 2 above)
Comment (CarmenMartinez) [reply src/gui/controllers/TaskController.java]: I still have doubts about how to do that.

**Review by AlexisGlez — APPROVED** [review]

**Pull request merged**